EventFilter/EcalRawToDigi/src/MatacqRawEvent.cc

```diff
@@ -80,10 +80,9 @@ const MatacqRawEvent::field32spec_t MatacqRawEvent::side32              = {13,0x
 
 void MatacqRawEvent::setRawData(const unsigned char* pData, size_t maxSize){
   error = 0;
-  unsigned char* begin = (unsigned char*) pData;
-  int16le_t* begin16 = (int16le_t*) pData;
+  const int16le_t *begin16 = (const int16le_t *) pData;
   uint32le_t* begin32 = (uint32le_t*) pData;
-  int16le_t* pData16 = begin16;
+  const int16le_t* pData16 = (const int16le_t *) begin16;
```

Comment:
The cast here is redundant and confusing because `begin16` is already of the type `const int16le_t *`. Please delete it:

`const int16le_t* pData16 = begin16;`